qemu/tests/virtio_port_hotplug.py

```diff
@@ -31,32 +32,43 @@ def run(test, params, env):
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
+    os_type = params["os_type"]
     timeout = int(params.get("login_timeout", 360))
-    if params["os_type"] == "windows":
-        run_bg_test(test, params, vm)
-    for repeat in range(int(params.get("repeat_times", 1))):
-        repeat += 1
+    module = params.get("modprobe_module")
+    unload_module = params.get("unload_module", "yes")
```

Comment:
`unload_module = params.get("unload_module", "yes") == 'yes'`

Reply:
unload_module = params.get_boolean("unload_module")

Reply:
@nanliu-r I would like to high light this conversation, use `get_boolean`.

Reply:
update.